### DISCUSSION TITLE
About.md page too wide

### DISCUSSION THREAD

**Discussion by likelymichael** [mmistakes/minimal-mistakes#3419] [Q&A] [answered]
Why is my about page so wide? Can't seem to figure it out. Manual setup by forking repo. Made all the updates from the guide, and then added about.md pages and a few others from the "easy template setup". Below is an example page, where the text goes from edge to edge. No other page does this, just about page.

![2022-02-04 16_59_15-Downloads](https://user-images.githubusercontent.com/8037512/152618700-66683d1a-a219-4d53-b33d-b4ca99fd260f.png)

**Comment by iBug** [comment, marked as answer]
You're most certainly setting `layout: post` when you should've set `layout: single`. Other than that there's nothing to diagnose unless you share a repository where we can take a look at.

**Reply by likelymichael** [reply]
> You're most certainly setting `layout: post` when you should've set `layout: single`.

Nope, definitely using single:

![2022-02-05 08_08_02-mclacore github io__config yml at master · mclacore_mclacore github io](https://user-images.githubusercontent.com/8037512/152647405-e0460eb1-3483-4ab3-8097-3472eb44cf4e.png)

Here's my repo: https://github.com/mclacore/mclacore.github.io

**Reply by mmistakes** [reply]
You’ve set front matter defaults for posts. Your about.md is a `page`.

Add `layout: single` to your [about.md](https://github.com/mclacore/mclacore.github.io/blob/master/_pages/about.md) page

**Comment by likelymichael** [comment]
Not really sure how to fix that. Change front matter default to page?

**Reply by mmistakes** [reply]
Add another scope for pages. Check the starter config for an example:

https://github.com/mmistakes/mm-github-pages-starter/blob/master/_config.yml#L90-L108
